python/dgl/graphbolt/impl/torch_based_feature_store.py

```diff
@@ -231,3 +234,40 @@ def pin_memory_(self):
         """In-place operation to copy the feature store to pinned memory."""
         for feature in self._features.values():
             feature.pin_memory_()
+
+    def __repr__(self) -> str:
+        return _torch_based_feature_store_str(self._features)
+
+
+def _torch_based_feature_str(feature: TorchBasedFeature) -> str:
+    final_str = "TorchBasedFeature("
+    indent_len = len(final_str)
+
+    def _add_indent(_str, indent):
+        lines = _str.split("\n")
+        lines = [lines[0]] + [" " * indent + line for line in lines[1:]]
+        return "\n".join(lines)
+
+    feature_str = f"feature={_add_indent(str(feature._tensor), indent_len + 8)}"
```

Comment:
what does `8` mean?

Reply:
the length of `feature=`

Reply:
then replace it with `len()` instead of magic number to be more readable.